### PR TITLE
linter issues fix

### PR DESCRIPTION
**Pull request by AdoAdoAdo**
fix some linter issues
give more time for test to pass libp2p module init.

### CODE REVIEW

**Review by sasurobert — APPROVED** [review]

File: factory/statusComponents_test.go [hunk 1]
```diff
@@ -276,6 +276,10 @@ func getCryptoComponents(coreComponents factory.CoreComponentsHolder) factory.Cr
 func getStateComponents(coreComponents factory.CoreComponentsHolder, shardCoordinator sharding.Coordinator) factory.StateComponentsHolder {
 	stateArgs := getStateArgs(coreComponents, shardCoordinator)
 	stateComponentsFactory, err := factory.NewStateComponentsFactory(stateArgs)
+	if err!=nil{
+		fmt.Println("getStateComponents NewStateComponentsFactory", "error", err.Error())
```
Comment (bogdan-rosianu) [inline factory/statusComponents_test.go:280]: log ?

**Review by BeniaminDrasovean — DISMISSED** [review]

File: node/node.go [hunk 2]
```diff
@@ -408,28 +406,30 @@ func (n *Node) printTxSentCounter(ctx context.Context) {
...
 	counterSeconds := 0
 
-	select {
-	case <-time.After(time.Second):
-		txSent := atomic.SwapUint32(&n.txSentCounter, 0)
-		if txSent > maxTxCounter {
-			maxTxCounter = txSent
-		}
-		totalTxCounter += uint64(txSent)
+	for {
```
Comment (bogdan-rosianu) [inline node/node.go:409]: so before this, the print was displayed only once?

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: process/coordinator/process_test.go [hunk 3]
```diff
@@ -2251,10 +2251,7 @@ func TestTransactionCoordinator_IsMaxBlockSizeReachedShouldWork(t *testing.T) {
 	argsTransactionCoordinator := createMockTransactionCoordinatorArguments()
 	argsTransactionCoordinator.BlockSizeComputation = &mock.BlockSizeComputationStub{
 		IsMaxBlockSizeWithoutThrottleReachedCalled: func(i int, i2 int) bool {
-			if i+i2 > 4 {
-				return true
-			}
-			return false
+			return i+i2 > 4
```
Comment (mateigraura) [inline process/coordinator/process_test.go:2254]: 👍🏻

**Review by mateigraura — DISMISSED** [review]

File: node/node.go (hunk 2 above)
Comment (AdoAdoAdo) [reply node/node.go:409]: yes, the refactor in this area was not correct.

File: factory/statusComponents_test.go (hunk 1 above)
Comment (AdoAdoAdo) [reply factory/statusComponents_test.go]: done

**Review by BeniaminDrasovean — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**